temporalcli/commandsgen/commands.yml

```diff
@@ -3333,3 +3304,37 @@ option-sets:
         description: |
           Run ID.
           If unset, updates the currently-running Workflow Execution.
+
+  - name: nexus-endpoint-identity
+    description: Nexus endpoint commands
+    options:
+      - name: name
+        type: string
+        description: Endpoint name.
+        required: true
+
+  - name: nexus-endpoint-config
+    description: Nexus endpoint commands
+    options:
+      - name: description
+        type: string
+        description: |
+          Nexus Endpoint description.
+          You may use Markdown formatting in the Nexus Endpoint description.
+      - name: description-file
+        type: string
+        description: |
+          Path to the Nexus Endpoint description file.
+          The contents of the description file may use Markdown formatting.
```

Comment:
If you mention JSON near 3323, then you can reduce confusion here by mentioning that the file does not need special formatting or JSON elements. Plain text or Markdown-formatted text alone are sufficient.

Reply:
It doesn't need to be a JSON string, you can do this:
```
temporal operator nexus endpoint create \
 13   --name myendpoint \
 14   --target-namespace my-target-namespace \
 15   --target-task-queue my-handler-task-queue \
 16   --description "## Sales Services
 17 Workflow's to support Customer-to-Order generation.
 18
 19 ## other
 20 stuff
 21 "
```

in practice `description` works well for simple descriptions.

if it gets too big a `description-file` is likely more suitable.

Reply:
Understood. (You'd still need line escapes)